BFWControls/Modules/Carousel/Controller/CarouselViewController.swift

```diff
@@ -44,6 +44,9 @@ class CarouselViewController: UICollectionViewController {
         return cellIdentifiers?.count ?? 0
     }
     
+    var pageControl = UIPageControl()
+    var collectionViewSize: CGSize?
```

Comment:
I'm currently using it in my subclass. I'm looking to replace that usage right now.